Makefile

```diff
@@ -0,0 +1,14 @@
+PYTHON ?= /usr/bin/python
+
+.PHONY: install
+install:
+	$(PYTHON) setup.py install
+
+.PHONY: clean
+clean:
+	$(PYTHON) setup.py clean --all
+
+.PHONY: test
+test:
+	pip install -r requirements.txt -r test/requirements.txt
+	$(PYTHON) -m pytest -vv
```

Comment:
To get the most out of the tests (ex: code coverage), use the shell scripts in ```test/utils``` to run the tests.